org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -237,10 +249,21 @@ public void initialize() {
         // If no key exists, generate a random key and save it back to the configuration
         if (!networkKey.isValid()) {
             networkKey = ZigBeeKey.createRandom();
-            logger.debug("Key initialised {}", networkKey);
+            logger.debug("Network key initialised {}", networkKey);
         }
 
-        logger.debug("Key final array {}", networkKey);
+        logger.debug("Network key final array {}", networkKey);
+
+        // Process the link key
+        try {
+            logger.debug("Link Key String {}", linkKeyString);
+            linkKey = new ZigBeeKey(linkKeyString);
+        } catch (IllegalArgumentException e) {
+            linkKey = KEY_ZIGBEE_ALLIANCE_O9;
+            logger.debug("Link Key String has invalid format. Revert to default key.", linkKeyString);
```

Comment:
I believe the `{}` placeholder for the `linkKeyString` parameter is missing in the log string.

Reply:
Thanks - cut and paste error ;)